plugins/out_s3/s3.c

```diff
@@ -1989,7 +1989,7 @@ static flb_sds_t flb_pack_msgpack_extract_log_key(void *out_context, const char
     }
 
     /* Throw error once per chunk if at least one log key was not found */
-    if (log_key_missing == FLB_TRUE) {
+    if (log_key_missing > 0) {
```

Comment:
FLB_TRUE is just an `int = 1` and this is a counter, so the current check will lead to the message not being printed if there are more than one missing log keys. @matthewfala